pkgs/games/r2modman/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, mkYarnPackage
+, fetchFromGitHub
+, gitUpdater
+, makeWrapper
+, makeDesktopItem
+, electron
+}:
+
+let
+  execName = "r2modman";
+in
+mkYarnPackage rec {
+  name = "r2modmann";
+  version = "3.1.41";
+
+  src = fetchFromGitHub {
+    owner = "ebkr";
+    repo = "r2modmanPlus";
+    rev = "v${version}";
+    hash = "sha256-nnmzIpV+Rn2zBJDBUehcUpvh3U4jqGLLGoIoxlZy6SA=";
+  };
+
+  packageJSON = ./package.json;
+  yarnLock = ./yarn.lock;
+  yarnNix = ./yarn.nix;
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  DISABLE_ESLINT_PLUGIN = "true";
+  # Workaround for webpack bug
+  # https://github.com/webpack/webpack/issues/14532
+  NODE_OPTIONS = "--openssl-legacy-provider";
+
+  postBuild =
+    let
+      yarnCmd = "yarn --offline --frozen-lockfile --ignore-engines --ignore-scripts --lockfile ${yarnLock}";
+    in
```

Comment:
Was there a compelling reason to put this in a let statement when it was only used once? I don't really mind all that much either way, I'm mostly just curious

Reply:
Partly leftover from initial attempts at getting it to build, and partly to make the script easier to read.